src/common/util.js

```diff
@@ -38,6 +38,10 @@ function getPenultimate(arr) {
   return null;
 }
 
+/**
+ * @param {string | RegExp} chars
+ * @returns {(text: string, index: number, opts?: object) => any}
```

Comment:
From a quick glance at the function body, doesn’t this return `number | false`? If so, the `skip*` functions below need to be updated as well.

Reply:
This is probably the reason for the errors you mention below.

Reply:
But what is correct? The website or the code?

Reply:
What website?

Reply:
https://prettier.io/docs/en/plugins.html#utility-functions

Reply:
The code.

Reply:
Oh okay 👍 

I will go to bed now 💤 
I can resolve this tomorrow